modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/designs_and_tech.dm

```diff
@@ -0,0 +1,202 @@
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_CARGO
+
+/datum/design/board/xeno_researcher
+	name = "Machine Design (Xenoarch Researcher)"
+	desc = "Allows for the construction of circuit boards used to build a new xenoarch researcher."
+	id = "xeno_researcher"
+	build_path = /obj/item/circuitboard/machine/xenoarch_researcher
+	category = list("Research Machinery")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/xeno_digger
+	name = "Machine Design (Xenoarch Digger)"
+	desc = "Allows for the construction of circuit boards used to build a new xenoarch digger."
+	id = "xeno_digger"
+	build_path = /obj/item/circuitboard/machine/xenoarch_digger
+	category = list("Research Machinery")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/xeno_scanner
+	name = "Machine Design (Xenoarch Scanner)"
+	desc = "Allows for the construction of circuit boards used to build a new xenoarch scanner."
+	id = "xeno_scanner"
+	build_path = /obj/item/circuitboard/machine/xenoarch_scanner
+	category = list("Research Machinery")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/xeno_recoverer
+	name = "Machine Design (Xenoarch Recoverer)"
+	desc = "Allows for the construction of circuit boards used to build a new xenoarch recoverer."
+	id = "xeno_recoverer"
+	build_path = /obj/item/circuitboard/machine/xenoarch_recoverer
+	category = list("Research Machinery")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/techweb_node/basic_xenoarch
+	id = "basic_xenoarch"
+	starting_node = TRUE
+	display_name = "Basic Xenoarchaeology"
+	description = "The basic designs of xenoarchaeology."
+	design_ids = list(
+		"hammercm1",
+		"hammercm2",
+		"hammercm3",
+		"hammercm5",
+		"hammercm10",
+		"bas_brush",
+		"xeno_tape",
+	)
+
+/datum/techweb_node/xenoarch_storage
+	id = "xenoarch_storage"
+	display_name = "Xenoarchaeology Storage"
+	description = "When dealing with xenoarchaeology, one may need storage."
+	prereq_ids = list("basic_xenoarch")
+	design_ids = list(
+		"xeno_belt",
+		"xeno_bag",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+
+/datum/techweb_node/xenoarch_machines
+	id = "xenoarch_machines"
+	display_name = "Xenoarchaeology Machines"
+	description = "Sometimes, xenoarchaeology can be time consuming, perhaps machines can help?"
+	prereq_ids = list("basic_xenoarch")
+	design_ids = list(
+		"xeno_researcher",
+		"xeno_digger",
+		"xeno_scanner",
+		"xeno_recoverer",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+
+/datum/techweb_node/adv_xenoarch
+	id = "adv_xenoarch"
+	display_name = "Advanced Xenoarchaeology"
+	description = "After some time, those tools we used have become antiquated-- we need an upgrade."
+	prereq_ids = list("basic_xenoarch", "xenoarch_machines", "xenoarch_storage")
+	design_ids = list(
+		"adv_hammer",
+		"adv_brush",
+		"adv_bag",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 3000)
```

Comment:
Your indenttation here is a little weird, are you sure it's using tabs with a width of 4 spaces, like everywhere else?